docs/guides/enroll-device.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Enroll a Device
+lang: en-US
+meta:
+  - name: keywords
+    content: >-
+      pomerium identity-access-proxy webauthn device id enroll
+      authentication authorization
+description: >-
+  This guide covers how to enroll a secure enclave device as a Pomerium end user.
+---
+
+# Enroll a Device
+
+If a Pomerium route is configured to [require device authentication](/docs/topics/ppl.md#device-matcher), then the user must register a [secure enclave device](/docs/topics/device-identity.md#authenticated-device-types) before accessing the route. Registration is easy, but different depending on the device being used to provide ID.
```

Comment:
The default device type is `any`, so not necessarily a secure enclave device.

Reply:
This relates to the outstanding question above; what other phrase can I use to describe the more general set of devices this applies to?

Reply:
My 2c is for "Any" to mean "Any Trusted Execution Environment Attestation".

Reply:
Adjusted in https://github.com/pomerium/pomerium/pull/2830/commits/88b9331976c1533d24b4605e99040a6d4727e020